pkgs/development/python-modules/mne/default.nix

```diff
@@ -18,26 +18,38 @@
   h5io,
   pymatreader,
   pythonOlder,
+  procps,
+  optipng,
 }:
 
 buildPythonPackage rec {
-  pname = "mne-python";
-  version = "1.9.0";
+  pname = "mne";
+  # https://github.com/mne-tools/mne-python/pull/13049 is required to build, it does not apply if fetchpatch'ed
+  stableVersion = "1.9.0";
+  version = "1.9.0-unstable-2025-05-01";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     owner = "mne-tools";
     repo = "mne-python";
-    tag = "v${version}";
-    hash = "sha256-Y9xkp1tji0+muP8+HqogRj5TGKCtpJzapKYWtmGw+F0=";
+    rev = "5df1721b488070e3b3928dface9dd0b8c39a3bef";
+    hash = "sha256-BCLejk0sVym+HRCfnTl5LTOGUMrQdxZbqhrCnIpzsvM=";
   };
 
+  env.SETUPTOOLS_SCM_PRETEND_VERSION = stableVersion;
+
   postPatch = ''
     substituteInPlace pyproject.toml  \
       --replace-fail "--cov-report=" ""  \
```

Comment:
nit: `pytest-cov-stub` would be a bit neater (unless it doesn't work for some reason).